src/data_vis.ts

```diff
@@ -20,10 +20,15 @@ import {WarningMessageManager} from "./WarningMessage";
 
 // stuff to handle the data set being split into multiple files
 const initTouchstone: string = "201710gavi-201907wue";
-export let addedDataSets: string[] = [];
-const update = appendToDataSet([initTouchstone], addedDataSets, []);
-addedDataSets = update.newSeenList;
-export let impactData = update.newDataSet;
+const montaguDataSets: DataSet[] = [
+    { name : "method_2", data : [], seen : [], prev: [] },
+    { name : "method_0", data : [], seen : [], prev: [] },
+    { name : "method_1", data : [], seen : [], prev: [] },
+];
+
+appendToDataSet([initTouchstone], "method_2", montaguDataSets, true);
```

Comment:
Why are some methods initialised with different touchstones?

Reply:
Method 2 has data for multiple touchstones, i set it to the latest.

Methods 0 and 1 only have data for one touchstone at the moment, so is set the default to that. This is also why jump between touchstone when changing method, keeping the same touchstone often leads to switching to a plot with no data.